assets/training/model_management/src/run_model_preprocess.py

```diff
@@ -86,15 +86,15 @@ def _validate_transformers_args(args):
     if not args.get("task"):
         raise Exception("task is a required parameter for hftransformers MLflow flavor.")
     task = args["task"]
-    if not SupportedTasks.has_value(task):
+    if not TransformersSupportedTasks.has_value(task):
```

Comment:
since we have validation under factory as well for task_type, we can remove checks here?

Reply:
Moved these validations to convertor base classes.